### PR TITLE
ELPP-2495 - Silent correction in sync with normal workflows

### PR DESCRIPTION
**Pull request by Jenniferstrej**

### CODE REVIEW

File: workflow/workflow_ProcessArticleZip.py [hunk 1]
```diff
@@ -112,6 +112,17 @@ def __init__(self, settings, logger, conn=None, token=None, decision=None,
                         "start_to_close_timeout": 60 * 5
                     },
                     {
+                        "activity_type": "CopyGlencoeStillImages",
```
Comment (giorgiosironi) [inline workflow/workflow_ProcessArticleZip.py]:
```
$ grep -rl CopyGlencoeStillImages workflow
workflow/workflow_CopyGlencoeStillImages.py
workflow/workflow_IngestArticleZip.py
workflow/workflow_SilentCorrectionsProcess.py
```
in how many places do we need this activity? I though it was only needed once per run?

File: workflow/workflow_SilentCorrectionsIngest.py [hunk 2]
```diff
@@ -101,6 +101,50 @@ def __init__(self, settings, logger, conn=None, token=None, decision=None,
                         "start_to_close_timeout": 60 * 15
                     },
                     {
+                        "activity_type": "ConvertImagesToJPG",
```
Comment (giorgiosironi) [inline workflow/workflow_SilentCorrectionsIngest.py]: ConvertImagesToJPG is consistent with `IngestArticleZip` workflow

File: workflow/workflow_SilentCorrectionsIngest.py [hunk 3]
```diff
@@ -101,6 +101,50 @@ def __init__(self, settings, logger, conn=None, token=None, decision=None,
...
+                        "version": "1",
+                        "input": data,
+                        "control": None,
+                        "heartbeat_timeout": 60 * 15,
+                        "schedule_to_close_timeout": 60 * 15,
+                        "schedule_to_start_timeout": 300,
+                        "start_to_close_timeout": 60 * 15
+                    },
+                    {
+                        "activity_type": "DepositIIIFAssets",
```
Comment (giorgiosironi) [inline workflow/workflow_SilentCorrectionsIngest.py]: DepositIIIFAssets is consistent with `IngestArticleZip` workflow too

File: workflow/workflow_SilentCorrectionsIngest.py [hunk 4]
```diff
@@ -101,6 +101,50 @@ def __init__(self, settings, logger, conn=None, token=None, decision=None,
...
+                        "version": "1",
+                        "input": data,
+                        "control": None,
+                        "heartbeat_timeout": 60 * 15,
+                        "schedule_to_close_timeout": 60 * 15,
+                        "schedule_to_start_timeout": 300,
+                        "start_to_close_timeout": 60 * 15
+                    },
+                    {
+                        "activity_type": "VerifyImageServer",
```
Comment (giorgiosironi) [inline workflow/workflow_SilentCorrectionsIngest.py]: same question as `CopyGlencoeStillImages`, what leads us to do this in both ingest and and process (may be necessary)

File: workflow/workflow_ProcessArticleZip.py (hunk 1 above)
Comment (Jenniferstrej) [inline workflow/workflow_ProcessArticleZip.py]: Yes, you are right, I missed the removal of that one. Changing now.

File: workflow/workflow_SilentCorrectionsIngest.py (hunk 4 above)
Comment (Jenniferstrej) [inline workflow/workflow_SilentCorrectionsIngest.py]: It's necessary just for now as the same images are copied again in DepositAssets. When I remove this redundancy I will also remove VerifyImageServer from ProcessArticleZip and SilentCorrectionsProcess

Comment (giorgiosironi) [inline workflow/workflow_SilentCorrectionsIngest.py]: I see this PR has been updated leaving ProcessArticleZip in the state we want it to be, but it doesn't have the removal of CopyGlencoeStillImages from SilentCorrectionsProcess (it's in there in `develop`)

Comment (Jenniferstrej) [inline workflow/workflow_SilentCorrectionsIngest.py]: Sorry, my bad. fixed.

**Pull request merged**